docs/kratos/concepts/security.mdx

```diff
@@ -3,210 +3,40 @@ id: security
-      // by GitHub.
-      [if "email_primary" in claims && claims.email_verified then "email" else null]: claims.email_primary,
-    },
-  },
-}
-```
-
-### Disabling account enumeration defenses
-
-Enforcing email verification, which requires an email round trip and disrupts the sign up process, isn't always feasible. In these
-cases, you might want to disable account enumeration defenses.
-
-If you enable the `session` post-registration workflow - signing the user in directly after registration - disables account
-enumeration defenses:
-
-```yaml
-selfservice:
-  flows:
-    registration:
-      after:
-        password:
-          - hook: session
-```
-
-## Account Takeover Defenses
-
-The Settings Flow implements account takeover defenses as it isn't possible to change the password without knowing the existing
-password. A good example of this flow is the
-[GitHub sudo mode](https://help.github.com/en/github/authenticating-to-github/sudo-mode).
-
-## Passwords
-
-Head over to the [password policy page](../../concepts/password-policy.mdx).
-
-## Anti-automation
-
-:::warning
-
-This feature is a work in progress and is tracked as [kratos#138](https://github.com/ory/kratos/issues/138).
-
-:::
-
-Actions that cause out-of-band communications, such as sending an activation link via email or an activation code via SMS, can be
-abused by automated systems. The goal of such an attack is to send out so many emails or SMS, that your reputation worsens (spam
-filters) or you're faced with massive costs (carrier fees).
-
-CAPTCHA renders these attacks either very difficult or impossible. CAPTCHA will be required in the following scenarios:
-
-- The user tries to register more than one account within 72 hours.
-- The user failed provide valid credentials for the third time within 12 hours.
-- The user tries to recover their account for the second time within 72 hours.
-
-For integration guidelines, please check the individual flow's (registration, login, account recovery) integration documentation.
-
-## Bruteforce attacks
-
-Will be addressed in a future release.
-
-## Phishing attacks
-
-Will be addressed in a future release.
-
-## Social engineering attacks
-
-Will be addressed in a future release.
-
-## SMS spoofing attacks
+## Digital identity guidelines
 
-Will be addressed in a future release.
+There is no universally accepted standard for digital identity.  
+Ory follows [Digital Identity Guidelines](https://pages.nist.gov/800-63-3/) established by the
+[National Institute of Standards and Technology (NIST)](https://www.nist.gov/).  
+These guidelines are accompanied by a [FAQ](https://pages.nist.gov/800-63-3/) that provides additional information and answers to
+common questions.
 
-## Account enumeration
+## Brute-force defenses
 
-:::warning
+Ory Network provides Ory Identities with protection against brute force attacks by rate-limiting requests to API public endpoints,
```

Comment:
```suggestion
Ory Network provides Ory Identities with protection against brute-force attacks by rate limiting requests to API public endpoints,
```
Adjective b4 noun = hyphen
rate limiting = no hyphen (I know Grammarly tells you to use a hyphen, but "rate limiting" here is not describing a quality of "requests")